### ISSUE TITLE
I'm logged in using Demo FastHub clientId

### ISSUE THREAD

**Issue by MiniGod** [k0shk0sh/FastHub#2262]
**FastHub Version:**  4.6.7
**Android Version:**  8.0.0
**Device Information:**  
- MANUFACTURER: Samsung
- BRAND: Samsung
- MODEL: S8 (SM-G950F)
---

Spotted this when looking at ["Authorized OAuth Apps"](https://github.com/settings/applications):

![image](https://user-images.githubusercontent.com/60520/52304596-0013c780-298b-11e9-8889-c027974d59ac.png)

> Demo FastHub clientId & key. If you see this in production, Please report it at https://github.com/k0shk0sh/FastHub

I've only used FastHub from Play Store.

After I logged out and back in with OAuth, "FastHub" appeared in the list of OAuth Apps (without "Demo"). There was no "FastHub" before, only "Demo FastHub". 
I think it's a _very_ long time since I logged in, so the credentials have probably been carried for a long time? I don't know how to see when the OAuth app was authorized. If you need any more info, please let me know.

**Comment by k0shk0sh** [comment]
Did u try FastHub- from here before? 

Also I didn't get the issue/problem in here could you please tell what's the issue/problem you are facing? 

Thanks

**Comment by MiniGod** [comment]
I've only used FastHub from Play Store. Somehow I've always been logged in with a OAuth App named "Demo FastHub".

The description of the app tells me to report it here (see screenshot and quote above), assuming that "production" the FastHub app on Play Store.

I don't know how I was logged in with that OAuth app. I'm only trying to help out here. Love FastHub!

TLDR: I don't have an issue/problem, but I was told to report this by the description of the OAuth app.

**Comment by MiniGod** [comment]
I assume you created that description of the "Demo FastHub" OAuth app, and that you did that because you wanted people to report it if accidentally was pushed to "production", i.e. play store.

Feel free to __close this issue__ if you see this as a non-issue.

**Comment by k0shk0sh** [comment]
That's kinda weird tbh as I'm almost certain that this never happened! I might have made mistakes but this ain't one 😂. 

I'll try and find out what's going on, if I got any info I'll let you know :). 

I created that key like Feb 2017 so trust me I don't remember haha, I'll have to look first.
